src/CommonDBTM.php

```diff
@@ -1420,7 +1420,7 @@ public function getLink($options = [])
 
         $link = $this->getLinkURL();
 
-        $label = $this->getNameID($options);
+        $label = \htmlspecialchars($this->getNameID($options));
```

Comment:
This one is far more complex. `getNameID` may return a HTML snipper or a raw name, depending on paramater. Changes made on this function should be done in a dedicated PR, as it may be really complex to validate.
```suggestion
        $label = $this->getNameID($options);
```